tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/variable-tests/Config.toml

```diff
@@ -0,0 +1,2 @@
+[variable_tests]
+    port = 9090
```

Comment:
Do we need to keep a tab here?

Reply:
Fixed with [7a49014](https://github.com/ballerina-platform/ballerina-lang/pull/28809/commits/7a490140dba6ff648098b8ef1f3162245729dfed)